lib/timers.js

```diff
@@ -481,6 +481,7 @@ Timeout.prototype.unref = function() {
     }
 
     var handle = reuse(this);
+    delete handle._list;
```

Comment:
> Is this what you had in mind by "Messes with some tests"?

Yeah, besides the fact that it is far more common and can more easily induce a perf problem.